runtimes/spiritnet/src/lib.rs

```diff
@@ -668,6 +670,23 @@ impl pallet_utility::Config for Runtime {
 
 impl pallet_randomness_collective_flip::Config for Runtime {}
 
+impl public_credentials::Config for Runtime {
```

Comment:
We are not constructing the pallet. IMO, we should still remove all `public_credentials` code right?

Reply:
9aa8086eaad7298a1a56b361fc1f93cdd98dfd74